php/providers/pecl.rb

```diff
@@ -102,20 +111,12 @@
 
 end
 
-def get_extension_dir
-  @extension_dir ||= begin
-    p = shell_out("#{new_resource.prefix}/bin/php-config --extension-dir")
-    p.stdout.strip
-  end
+def get_extension_dir(prefix)
+  config = ::Php::Config.new('', {})
+  config.get_extension_dir(prefix)
 end
 
 def get_extension_files(name)
-  files = []
-
-  p = shell_out("pecl list-files #{name}")
-  p.stdout.each_line.grep(/^src\s+.*\.so$/i).each do |line|
-    files << line.split[1]
-  end
-
-  files
+  config = ::Php::Config.new(new_resource.name, {})
```

Comment:
Same here, this let me then reduce the dependency of the LWRP and I moved it to `Php::Config`